src/services/initializer-service.ts

```diff
@@ -112,15 +115,19 @@ export class InitializerService {
 
     this.createDefaultDirectories()
 
-    const bundleDescriptorService = new BundleDescriptorService(
-      this.filesys.getBundleDirectory()
-    )
-    const descriptor = bundleDescriptorService.getBundleDescriptor()
-    bundleDescriptorService.writeBundleDescriptor({
-      ...descriptor,
-      name,
-      version
-    })
+    try {
+      const bundleDescriptorService = new BundleDescriptorService(
+        this.filesys.getBundleDirectory()
+      )
+      const descriptor = bundleDescriptorService.getBundleDescriptor()
+      bundleDescriptorService.writeBundleDescriptor({
+        ...descriptor,
+        name,
+        version
+      })
+    } catch {
+      throw new CLIError(ERROR_NO_BUNDLE_DESCRIPTOR)
```

Comment:
It is also possible to use `Errors.warn(color.bold.yellow(ERROR_NO_BUNDLE_DESCRIPTOR))` (`Errors` from `'@oclif/core'`), that can be used in services instead of the `this.warn` available in commands classes.
In this way no exception is thrown, avoiding the try/catch+if in the command, and the message is simply print on the stderr.

Reply:
done